backend/plugins/frontline_api/src/modules/integrations/call/db/models/Integrations.ts

```
@@ -0,0 +1,67 @@
+    userId: string,
+    integrationId?: string,
+  ): Promise<ICallIntegrationDocument>;
+}
+
+export const loadCallIntegrationClass = (models: IModels) => {
+  class Integration {
+    public static async getIntegrations(
+      userId: string,
+      integrationId?: string,
+    ) {
+      let integrations = await models.CallIntegrations.find({
```

ESLint

[error] 'integrations' is never reassigned. Use 'const' instead.

(prefer-const)